requirements/base.in

```diff
@@ -10,3 +10,6 @@ stevedore                 # Used by plugins to make importing easier
 django-crum               # Used by logging filters
 PyNaCl                    # User-friendly cryptography (wrapper and bindings for libsodium)
 click                     # CLI tools
+opentelemetry-distro                      # OpenTelemetry SDK, API, and instrumentation
+opentelemetry-instrumentation-django      # Django-specific OpenTelemetry instrumentation
+opentelemetry-exporter-otlp-proto-grpc    # Export OpenTelemetry to a collector via GRPC
```

Comment:
Author's note: We might want to skip the gRPC dependency unless we're comfortable saying that this will be the canonical protocol to use for OpenTelemetry in Open edX. (The other main option is http/protobuf.)